examples/python/interface.py

```diff
@@ -142,7 +142,7 @@ def control_test(control_module='', start_time=0, warmup_period=0, length=24*360
         # Record test simulation start time
         start_time = int(res['time'])
         # Set final time and total time steps to be very large since scenario defines length
-        final_time = np.inf
+        final_time = 10e9 # np.inf
```

Comment:
I met the error message "TypeError: Object of type float is not JSON serializable" when attempting to directly pass np.inf to the json parameter